src/System.Data.Common/tests/System/Data/DataTableTest.cs

```diff
@@ -1429,6 +1358,7 @@ public void ImportRowTypeChangeTest()
 
                     if (invalid.Contains(Tuple.Create(types[a], types[b])))
                     {
+                        //TODO: what
                         try
```

Comment:
why is https://github.com/dotnet/corefx/pull/39958/files#diff-e6b7d1cdfeb7009e84339b8fba17ee29R1362-R1370 swallowing every exception?